app/src/main/java/com/asfoundation/wallet/ui/iab/OnChainBuyPresenter.kt

```diff
@@ -154,10 +154,10 @@ class OnChainBuyPresenter(private val view: OnChainBuyView,
       Payment.Status.FORBIDDEN -> Completable.fromAction { handleFraudFlow() }
           .andThen(onChainBuyInteract.remove(transaction.uri))
 
-      Payment.Status.ERROR -> Completable.fromAction { showError(null) }
+      Payment.Status.ERROR -> Completable.fromAction { showError(null, "Payment status error") }
```

Comment:
Can't we send the status as a string to rakam? Instead of having hardcoded text we could just send "ERROR" or "FORBIDDEN" or "UNKNOWN"